src/app/hooks/useGlean.ts

```diff
@@ -17,16 +17,25 @@ export const useGlean = () => {
     // Enable upload only if the user has not opted out of tracking.
     const uploadEnabled = navigator.doNotTrack !== "1";
 
+    const channel = process.env.NEXT_PUBLIC_APP_ENV;
+    if (!channel) {
+      console.warn("No channel defined in env var NEXT_PUBLIC_APP_ENV");
+    }
+
     Glean.initialize("monitor.frontend", uploadEnabled, {
       // This will submit an events ping every time an event is recorded.
       maxEvents: 1,
+      channel,
     });
 
     // Glean debugging options can be found here:
     // https://mozilla.github.io/glean/book/reference/debug/index.html
     if (process.env.NEXT_PUBLIC_APP_ENV === "local") {
       // Enable logging pings to the browser console.
       Glean.setLogPings(true);
+      // Tag pings for the Debug Viewer
+      // @see https://debug-ping-preview.firebaseapp.com/pings/fx-monitor-local-dev
+      Glean.setDebugViewTag("fx-monitor-local-dev");
```

Comment:
Imma confused.
is... do we need a different value for stage/prod, or does everything get stored in a Glean bucket named `*-local-dev`?
If it is different, should we toss this into an env var and add to .env-dist?